### DISCUSSION TITLE
v2/guide/markdown/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#365] [Announcements]
# v2/guide/markdown/

Besides Markdown syntaxes VuePress itself adds, vuepress-theme-hope enables more syntax in Markdown via [vuepress-plugin-md-enhance][md-enhance] plugin.

https://vuepress-theme-hope.github.io/v2/guide/markdown/

**Comment by shinylink-manual** [comment]
1
